apps/crn-server/src/data-providers/contentful/users.data-provider.ts

```diff
@@ -130,10 +130,13 @@ export class UserContentfulDataProvider implements UserDataProvider {
     const environment = await this.getRestClient();
     const user = await environment.getEntry(id);
     const result = await patchAndPublish(user, fields);
-    await this.waitForUpdated(id, result.sys.publishedVersion || Infinity);
+    await this.waitForUpdated(id, result.sys.publishedVersion);
   }
 
-  private async waitForUpdated(id: string, version: number) {
+  private async waitForUpdated(id: string, version: number | undefined) {
```

Comment:
```suggestion
  private async waitForUpdated(id: string, version?: number) {
```
though to be fair im still not a fan of an optional parameter only to satisfy typescript. I think it should just be 

```suggestion
  private async waitForUpdated(id: string, version: number) {
```